FabricExample/ios/Podfile

```diff
@@ -7,7 +7,6 @@ require Pod::Executable.execute_command('node', ['-p',
 
 platform :ios, min_ios_version_supported
 prepare_react_native_project!
-ENV['RCT_NEW_ARCH_ENABLED'] = '1'
```

Comment:
I would leave it explicit - but won't insist here.

Reply:
So I would go with explicte in podfile and skip the command one (I think it's overrided anyway)

Reply:
Changed in: https://github.com/software-mansion/react-native-screens/pull/2407/commits/5d6718bcd725abf90c185d5a83f114fc6db98725